cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -166,9 +166,9 @@ static Bom addMetadataProperty(final String name, final String value, final Stri
         return bom;
     }
 
-    static Bom addComponent(final String compName, final String name, final String value, final String description, final String fileName) {                    // Method to store Component --> name
+    static Bom addComponent(final String compName, final String name, final String value, final String description, final String fileName) {      // Method to store Component --> name & single name-value pair
         Bom bom = readJSONfile(fileName);
-    	Component comp = new Component();
+        Component comp = new Component();
```

Comment:
I think from your comments, we need top remove the "prop2" setting in addComponent ?
At the moment the example in build.xml for --addComponent adds an empty property.